File: server/vueblog-common/src/pojo/backup.rs
use serde::{Deserialize, Serialize};

#[derive(Debug, Serialize, Deserialize, Clone)]
pub struct SelectBackUp {
    pub username: String,
    pub password: String,
    pub operator: String,
    pub operator_password: String,
    pub bucket_name: String,
}

impl SelectBackUp {
    pub fn new() -> Self {
        Self {
            username: String::new(),
            password: String::new(),
            operator: String::new(),
            operator_password: String::new(),
            bucket_name: String::new(),
        }
    }
}

File: new_server/vueblog-common/src/main/java/cn/smilex/vueblog/common/entity/other/BackUp.java
package cn.smilex.vueblog.common.entity.other;

import com.fasterxml.jackson.annotation.JsonAlias;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.databind.annotation.JsonSerialize;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author smilex
 * @date 2022/12/17/13:21
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class BackUp {
    @JsonProperty("username")
    private String userName;

    @JsonProperty("password")
    private String passWord;

    @JsonProperty("operator")
    private String operator;

    @JsonProperty("operator_password")
    private String operatorPassWord;

    @JsonProperty("bucket_name")
    private String bucketName;
}
